elasticdl/python/master/master.py

```diff
@@ -220,6 +222,7 @@ def prepare(self):
             self.instance_manager.update_status(InstanceManagerStatus.PENDING)
             if self.distribution_strategy == DistributionStrategy.ALLREDUCE:
                 # Exposes the consensus service for allreduce-based training
+                self.rendezvous_server.start()
                 self.instance_manager.start_ftlib_consensus_service()
```

Comment:
Are we starting FTLib service as well in the meantime?

Reply:
We are developing AllReduce using Horovod, so we reserve FTLib service for the moment.